scheduler/executor/interfaces/go_cron_wrapper.go

```diff
@@ -0,0 +1,25 @@
+package interfaces
+
+import (
+	"context"
+	"github.com/flyteorg/flyteadmin/scheduler/repositories/models"
+	"time"
+)
+
+
+type RegisterFuncRef func(ctx context.Context, schedule models.SchedulableEntity, scheduledTime time.Time)
+
+// GoCronWrapper Wrapper interface to the gogf framework which is used for scheduler. It also locks in schedule time.
```

Comment:
wrong docs. Also a little confused, what are the 2 interfaces?
GoCronJobWrapper and GoCronWrapper